test/18-startup.spec.ts

```diff
@@ -52,5 +58,24 @@ describe('Startup', () => {
 		expect(anySupervisor.logger).to.not.be.null;
 		expect(anySupervisor.deviceState).to.not.be.null;
 		expect(anySupervisor.apiBinder).to.not.be.null;
+
+		const macAddresses: string[] = [];
+		reportCurrentStateStub.getCalls().forEach((call) => {
+			const m: string = call.args[0]['mac_address'];
+			if (!m || m === '') {
```

Comment:
This is equivalent to
```suggestion
			if (!m) {
```